### PR TITLE
Missing translations for Greek (el)

### LINKED ISSUE #38743: Missing translations for Greek (el)
Hello,

There are some missing translation for Greek (el). These should be added in branch 4.4. 
See https://github.com/symfony/symfony/issues/38710 for details and [this page](https://symfony-translations.nyholm.tech/#pr) for an example.

These are the files that should be updated: 
- [src/Symfony/Component/Form/Resources/translations/validators.el.xlf](https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Form/Resources/translations/validators.el.xlf)
- [src/Symfony/Component/Security/Core/Resources/translations/security.el.xlf](https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Security/Core/Resources/translations/security.el.xlf)
- [src/Symfony/Component/Validator/Resources/translations/validators.el.xlf](https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Validator/Resources/translations/validators.el.xlf)

NOTE: If you want to work on this issue, add a comment to assign it to yourself and let others know that this is already taken.

### PR DESCRIPTION
**Pull request by pavlakis**
| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | yes
| New feature?  | no 
| Deprecations? | no 
| Tickets       | Fix #38743. 
| License       | MIT
| Doc PR        | N/A

### CODE REVIEW

File: src/Symfony/Component/Form/Resources/translations/validators.el.xlf
Comment (Nyholm) [inline src/Symfony/Component/Form/Resources/translations/validators.el.xlf]: Oops, you missed this

File: src/Symfony/Component/Form/Resources/translations/validators.el.xlf
Comment (Nyholm) [inline src/Symfony/Component/Form/Resources/translations/validators.el.xlf]: Here too

File: src/Symfony/Component/Form/Resources/translations/validators.el.xlf
Comment (Nyholm) [inline src/Symfony/Component/Form/Resources/translations/validators.el.xlf]: Can you double check if this is correct?

File: src/Symfony/Component/Form/Resources/translations/validators.el.xlf
Comment (Nyholm) [inline src/Symfony/Component/Form/Resources/translations/validators.el.xlf:95]: Can you also double check this one please?

File: src/Symfony/Component/Form/Resources/translations/validators.el.xlf
Comment (Nyholm) [inline src/Symfony/Component/Form/Resources/translations/validators.el.xlf]: Does this say "URL" or "link"? It is technically a difference. Please make sure it says URL.

File: src/Symfony/Component/Security/Core/Resources/translations/security.el.xlf
Comment (Nyholm) [inline src/Symfony/Component/Security/Core/Resources/translations/security.el.xlf]: Can you check if this is correct too?

File: src/Symfony/Component/Validator/Resources/translations/validators.el.xlf
Comment (Nyholm) [inline src/Symfony/Component/Validator/Resources/translations/validators.el.xlf]: Does this say "code" or "password"?

**Review by Nyholm — COMMENTED** [review]
Thank you!

I've used Google translate to try to confirm this. I might be very wrong with my comments. But can you check those?

**Comment by Nyholm** [comment]
Hey @pavlakis, could you check my comments?

**Comment by pavlakis** [comment]
Thanks @Nyholm. Some of these make sense. I'll amend.

File: src/Symfony/Component/Form/Resources/translations/validators.el.xlf
Comment (pavlakis) [reply src/Symfony/Component/Form/Resources/translations/validators.el.xlf:95]: This is fine. I don't see how/why it should change.

File: src/Symfony/Component/Validator/Resources/translations/validators.el.xlf
Comment (pavlakis) [reply src/Symfony/Component/Validator/Resources/translations/validators.el.xlf]: Good spot! It is missing a word for the meaning to be `password`.

File: src/Symfony/Component/Form/Resources/translations/validators.el.xlf
Comment (Nyholm) [reply src/Symfony/Component/Form/Resources/translations/validators.el.xlf:95]: It is probably Google Translate that is wrong. =) It translated this to "Prices do not match". 

If you are happy with this line, then so am I.

File: src/Symfony/Component/Validator/Resources/translations/validators.el.xlf
Comment (Nyholm) [reply src/Symfony/Component/Validator/Resources/translations/validators.el.xlf]: Thank you

**Comment by Nyholm** [comment]
Awesome. Thank you. 

I just have one more comment. If you confirm that again then Im happy to merge.

File: src/Symfony/Component/Form/Resources/translations/validators.el.xlf
Comment (nicolas-grekas) [reply src/Symfony/Component/Form/Resources/translations/validators.el.xlf:95]: maybe `Οι αξίες δεν ταιριάζουν.`?

Comment (pavlakis) [reply src/Symfony/Component/Form/Resources/translations/validators.el.xlf:95]: It is lost in translation a bit, tbh. 

A word can mean different things depending on the context. So yes, it can mean `price` and `value` depending on the context.

I still think `τιμές` is the correct word to use here. This is also the same word used in previous translations too.

Comment (Nyholm) [reply src/Symfony/Component/Form/Resources/translations/validators.el.xlf:95]: Okey. Im 👍 for constancy. 


Thank you again.

**Review by Nyholm — APPROVED** [review]
I am super happy with this. 

Thank you

**Review by Nyholm — APPROVED** [review]
I changed the base to 4.4

(A few days ago, when the PR was created, 3.4 was the correct branch)


----


Congratulations to your first contribution. Not it is just to wait for another core team member to review and merge this.

**Comment by derrabus** [comment]
Thank you @pavlakis.

**Pull request merged**
